brickstrap-argv.sh

```diff
@@ -0,0 +1,237 @@
+#!/bin/bash
+#
+# This file is part of brickstrap.
```

Comment:
I dunno. The downside of something like Python is that it doesn't buy you much for the actual work of brickstrap which is mostly doing things like `cat`/`cp`,`ln` and globbing with `*` apart from invoking the multistrap and user-unshare scripts.